spec/support/utilities.rb

```diff
@@ -6,6 +6,20 @@ def valid_signin(user)
   click_button "Sign in"
 end
 
+def sign_in(user, options={})
+  if options[:no_capybara]
+    # Capybaraを使用していない場合にもサインインする。
+    remember_token = User.new_remember_token
+    cookies[:remember_token] = remember_token
+    user.update_attribute(:remember_token, User.encrypt(remember_token))
+  else
+    visit signin_path
+    fill_in "Email",    with: user.email
+    fill_in "Password", with: user.password
+    click_button "Sign in"
```

Comment:
[rubocop] Prefer single-quoted strings when you don't need string interpolation or special symbols.